feedbacks/tests/test_views.py

```diff
@@ -0,0 +1,255 @@
+from ddt import ddt, data
+from django.test import TestCase
+from django.urls import reverse
+
+from feedbacks.tests_data.test_view_data import PAGE_STRING_VALUES
+from feedbacks.tests_data.test_form_data import (
+    FEEDBACK_MIN_VALUE_IS_GREATER_MAX_VALUE_DATA,
+)
+from feedbacks.tests.mixins import (FeedbackCRUDViewsSetUpMixin,
+                                    RecommendationCRUDViewsSetUpMixin,
+                                    SetUpUserMixin, USERNAME, PASSWORD)
+from feedbacks.models import Feedback
+
+__all__ = ['FeedbackListViewTest', ]
+
+
+@ddt
+class FeedbackListViewTest(SetUpUserMixin, TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        for i in range(10):
+            Feedback.objects.create(feedback='Some text', min_value=i,
+                                    max_value=i + 5, indicator='PDI')
+
+    def test_redirect_if_not_logged_in(self):
+        response = self.client.get(reverse('feedback-list'), follow=True)
+        self.assertRedirects(response,
+                             f'/login/?next={reverse("feedback-list")}')
+        response = self.client.post(reverse('feedback-list'), follow=True)
+        self.assertRedirects(response,
+                             f'/login/?next={reverse("feedback-list")}')
+
+    def test_call_view_loads(self):
+        self.client.login(username=USERNAME, password=PASSWORD)
+        response = self.client.get(reverse('feedback-list'))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'feedbacks/feedback_list.html')
+
+    @data(*range(-10, 1))
+    def test_call_view_where_page_number_less_then_1(self, page):
+        self.client.login(username=USERNAME, password=PASSWORD)
+        response = self.client.get(f'{reverse("feedback-list")}?page={page}')
+        expected_response = self.client.get(
+            f'{reverse("feedback-list")}?page=1')
+        self.assertEqual(response.status_code, expected_response.status_code)
+        self.assertEqual(response.rendered_content,
+                         expected_response.rendered_content)
+
+    @data(*range(10, 21))
+    def test_call_view_where_page_number_is_to_large(self, page):
+        self.client.login(username=USERNAME, password=PASSWORD)
+        response = self.client.get(f'{reverse("feedback-list")}?page={page}')
```

Comment:
It looks that you using this method many times
https://github.com/IF-093-Python/CultureAnalyzer/blob/0992c02e7c32fc1ef1a5697928963be0b70b9271/feedbacks/tests/test_views.py#L54

You can to create function like this:
``` python 
def get_feedback_page(self, page):
    return self.client.get(f'{reverse("feedback-list")}?page={page}')
```
And then use its in the tests
``` python 
    response = self.get_feedback_page(page)
    expected_response = self.get_feedback_page(1)
    self.assertEqual(response.status_code, expected_response.status_code)
```